### PR TITLE
Enabling IQE tests run on pr_check RHIROS-1022

### PR DESCRIPTION
**Pull request by dgaikwad**
- Enabling "smoke" marker where we will be running some API tests

### CODE REVIEW

File: clowdapp.yaml [hunk 1]
```diff
@@ -10,6 +10,8 @@ objects:
     name: ros-ocp-backend
   spec:
     envName: ${ENV_NAME}
+    testing:
+      iqePlugin: ros_ocp
```
Comment (ntkathole) [inline clowdapp.yaml:14]: s/ros_ocp/ros-ocp

https://consoledot.pages.redhat.com/docs/dev/operating-your-app/testing-iqe/testing.html#_clowdapp_settings 

The name of the plugin you use here should be the "short name" of your plugin, and should match the name of your plugin’s image tag in quay.io

**Review by ntkathole — APPROVED** [review]
Looks good to me!
However if you want, you can consider enhancement like adding `source $CICD_ROOT/post_test_results.sh` at the end. This will add the Ibutsu URL and test run IDs as a git check on PRs.

File: pr_check.sh [hunk 2]
```diff
@@ -19,9 +23,12 @@ source $CICD_ROOT/build.sh
...
-cat << EOF > ${WORKSPACE}/artifacts/junit-dummy.xml
-<testsuite tests="1">
-    <testcase classname="dummy" name="dummytest"/>
-</testsuite>
-EOF
+# Run iqe-ros-ocp smoke tests with ClowdJobInvocation
+source $CICD_ROOT/cji_smoke_test.sh
+
+#mkdir -p $WORKSPACE/artifacts
+#cat << EOF > ${WORKSPACE}/artifacts/junit-dummy.xml
```
Comment (r14chandra) [inline pr_check.sh:30]: @dgaikwad I think you can remove these comments from the file.

**Comment by dgaikwad** [comment]
> Looks good to me! However if you want, you can consider enhancement like adding `source $CICD_ROOT/post_test_results.sh` at the end. This will add the Ibutsu URL and test run IDs as a git check on PRs.

Thank you

**Comment by dgaikwad** [comment]
> Hello @dgaikwad - is anything pending for this pull-request? Tests are not green!

Now everything is done from me.

**Review by saltgen — APPROVED** [review]
LGTM

**Pull request merged**